src/server/database/migrations/20210629030633-modify-users-to-checkers-association-add-owners-column.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.addColumn('usersToCheckers', 'isOwner', {
+      type: Sequelize.BOOLEAN,
+      allowNull: false,
+      defaultValue: true,
+    })
+  },
+
+  down: async (queryInterface, _) => {
+    // return queryInterface
+    //   .bulkDelete('usersToCheckers', { isOwner: false })
+    //   .then(() => {
+    //     queryInterface.removeColumn('usersToCheckers', 'isOwner')
+    //   })
```

Comment:
to delete?

Reply:
Good catch! @rlyss just to be clear, in the event of a rollback, the behaviour we want is such that all users (regardless of owner or not) would preserve access to the checker right? Do try to test this scenario out also.